client/src/modules/Opportunities/components/EditCv/index.tsx

```diff
@@ -47,9 +46,9 @@ export const EditCV = (props: Props) => {
   const visibleCoursesFormRef: RefObject<FormInstance> = createRef();
 
   const submitData = async (data: AllUserCVData) => {
-    const newData = await cvService.saveResumeData(data);
+    const newData = await service.saveResume(props.githubId, { data });
 
-    const { userData, contacts, visibleCourses } = splitDataForForms(newData);
+    const { userData, contacts, visibleCourses } = splitDataForForms(newData.data as AllUserCVData);
```

Comment:
curious why do u need casting: shouldn't it be same type?

Reply:
this is due to incompatibility of nullable enums - I tried various approaches but it doesn't set enum | null in client api